src/main/java/com/cleanroommc/groovyscript/compat/mods/aether_legacy/Accessory.java

```diff
@@ -0,0 +1,112 @@
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.cleanroommc.groovyscript.registry.ReloadableRegistryManager;
+import com.gildedgames.the_aether.api.accessories.AccessoryType;
+import com.gildedgames.the_aether.api.accessories.AetherAccessory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
+public class Accessory extends ForgeRegistryWrapper<AetherAccessory> {
+
+    private static final AccessoryType[] VALUES = AccessoryType.values();
+
+    public Accessory() {
+        super(GameRegistry.findRegistry(AetherAccessory.class), Alias.generateOf("accessory"));
+    }
+
+    public void add(AetherAccessory accessory) {
+        if (accessory != null) {
+            ReloadableRegistryManager.addRegistryEntry(this.getRegistry(), accessory);
+        }
+    }
+
+    @MethodDescription(type = MethodDescription.Type.ADDITION)
+    public void add(ItemStack item, String type) {
+        AccessoryType accessoryType = AccessoryType.MISC;
+        for (AccessoryType value : VALUES) {
+            if (value.getDisplayName().equalsIgnoreCase(type)) {
+                accessoryType = value;
+            }
+        }
+        AetherAccessory accessory = new AetherAccessory(item, accessoryType);
+        add(accessory);
+    }
+
+    public boolean remove(AetherAccessory accessory) {
+        if (accessory == null) return false;
+        ReloadableRegistryManager.removeRegistryEntry(this.getRegistry(), accessory.getRegistryName());
+        return true;
+    }
+
+    @MethodDescription(description = "groovyscript.wiki.removeByInput", example = @Example("item('aether_legacy:iron_pendant')"))
+    public void removeByInput(ItemStack input) {
+        this.getRegistry().getValuesCollection().forEach(accessory -> {
+            if (accessory.getAccessoryStack().isItemEqual(input)) {
+                ReloadableRegistryManager.removeRegistryEntry(this.getRegistry(), accessory.getRegistryName());
+            }
+        });
+    }
+
+    @MethodDescription(description = "groovyscript.wiki.removeAll", priority = 2000, example = @Example(commented = true))
+    public void removeAll() {
+        this.getRegistry().getValuesCollection().forEach(accessory -> {
+            ReloadableRegistryManager.removeRegistryEntry(this.getRegistry(), accessory.getRegistryName());
+        });
+    }
+
+    @MethodDescription(description = "groovyscript.wiki.streamRecipes", type = MethodDescription.Type.QUERY)
+    public SimpleObjectStream<AetherAccessory> streamEntries() {
+        return new SimpleObjectStream<>(this.getRegistry().getValuesCollection()).setRemover(this::remove);
+    }
+
+    @RecipeBuilderDescription(example = @Example(".input(item('minecraft:shield')).accessoryType('shield')"))
+    public RecipeBuilder recipeBuilder() {return new RecipeBuilder();}
```

Comment:
should be on separate lines (same with the other `recipeBuilder()`s). 
```suggestion
    public RecipeBuilder recipeBuilder() {
        return new RecipeBuilder();
}